doc/development/deprecations.rst

```diff
@@ -110,8 +116,7 @@ Completed deprecation cycles
   - Deprecated in v0.40
   - Removed in v0.41
 
-* The ``QNode.get_best_method`` and ``QNode.best_method_str`` methods have been removed. 
-  Instead, use the ``qml.workflow.get_best_diff_method`` function. 
+* The ``QNode.get_best_method`` and ``QNode.best_method_str`` methods have been Controlled
```

Comment:
```suggestion
* The ``QNode.get_best_method`` and ``QNode.best_method_str`` methods have been removed. 
  Instead, use the ``qml.workflow.get_best_diff_method`` function. 
```

Reply:
Yeah this place is ghosting since I could never saw this diff on my local. Let me try commit here and see what will happen